spring-xd-dirt/src/main/java/org/springframework/xd/dirt/stream/completion/StacktraceFingerprintingCompletionRecoveryStrategy.java

```diff
@@ -90,15 +91,12 @@ private void initFingerprints() {
 					// not influenced by the kind of parse. Use stream for now
 					this.parser.parse("__dummy", sample, ParsingContext.partial_stream);
 				}
-				catch (Exception exception) {
```

Comment:
Not sure about this one. It used to be able to catch (and compute stacktrace fingerprint) of checked exceptions, and now doesn't